src/integration-test/tests/citizen/claim/steps/claim.ts

```diff
@@ -115,7 +115,7 @@ export class ClaimSteps {
       case PartyType.INDIVIDUAL:
         partyTypePage.selectIndividual()
         individualDetailsPage.enterName(defendant.name)
-        individualDetailsPage.enterAddressOnPostCodeLookUp(postCodeLookup)
+        individualDetailsPage.enterAddressOnPostCodeLookUp(postcodeLookupQuery)
```

Comment:
Please rename to be `lookupAddress` as well to make it consistent.

Reply:
done